shared/js/media/media_frame.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-/* global Downsample, VideoPlayer */
+/* global Downsample, VideoPlayer, MediaUtils */
```

Comment:
On second thought, MediaUtils.formatDate() is such a trivial function that maybe it would be better to just reimplement it here rather than adding  a new dependency to this module.